jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -146,15 +148,15 @@ public void testTracerTagslist() {
   public void testTracerTagslistFormatError() {
     System.setProperty(Configuration.JAEGER_SERVICE_NAME, "Test");
     System.setProperty(Configuration.JAEGER_TAGS, "testTag1, testTag2 = testValue2");
-    io.jaegertracing.Tracer tracer = (io.jaegertracing.Tracer) Configuration.fromEnv().getTracer();
+    JaegerTracer tracer = (JaegerTracer) Configuration.fromEnv().getTracer();
```

Comment:
TBD - I think configuration should be returning JaegerTracer. But it's not part of this PR.